tests/testthat/test-read-fwf.R

```diff
@@ -127,6 +127,43 @@ test_that("error on empty spec (#511, #519)", {
   expect_error(read_fwf(txt, pos), "Zero-length.*specifications not supported")
 })
 
+# fwf_cols
+test_that("fwf_cols produces correct fwf_positions object with elements of length 2", {
+  expected <- fwf_positions(c(1,  9, 4),
```

Comment:
Can you please fix the indenting here too?

If the arguments don't fit on one line it should look like:

```
function_name(
   arg1,
   argument_name = arg2,
   ...
)
```